website/content/en/preview/tasks/managing-amis.md

```diff
@@ -77,6 +77,10 @@ For example, you could have:
   spec:
```

Comment:
Linking the line here since I can't comment on it. Seems like the Paragraph above this example needs to be updated: 
```
Test clusters: On lower environment clusters, you can run the latest AMIs for your workloads in a safe environment. The EC2NodeClass for these clusters could be set with a chosen amiFamily, but no amiSelectorTerms set. For example, the NodePool and EC2NodeClass could begin with the following:
```
ref: https://github.com/aws/karpenter-provider-aws/blob/main/website/content/en/preview/tasks/managing-amis.md#option-1-manage-how-amis-are-tested-and-rolled-out